components/index.ts

```diff
@@ -200,67 +199,67 @@ export {
     // setBasePath helper to use the UI behind proxies etc
     setBasePath,
     // Standard shoelace components
-    SlAlert,
-    SlAnimatedImage,
-    SlAnimation,
-    SlAvatar,
-    SlBadge,
-    SlBreadcrumb,
-    SlBreadcrumbItem,
-    SlButton,
-    SlButtonGroup,
-    SlCard,
-    SlCarousel,
-    SlCarouselItem,
-    SlCheckbox,
-    SlColorPicker,
-    SlCopyButton,
-    SlDetails,
-    SlDialog,
-    SlDivider,
-    SlDrawer,
-    SlDropdown,
-    SlFormatBytes,
-    SlFormatDate,
-    SlFormatNumber,
-    SlIcon,
-    SlIconButton,
-    SlImageComparer,
-    SlInclude,
-    SlInput,
-    SlMenu,
-    SlMenuItem,
-    SlMenuLabel,
-    SlMutationObserver,
-    SlOption,
-    SlPopup,
-    SlProgressBar,
-    SlProgressRing,
-    SlQRCode,
-    SlRadio,
-    SlRadioButton,
-    SlRadioGroup,
-    SlRange,
-    SlRating,
-    SlRelativeTime,
-    SlResizeObserver,
-    SlSelect,
-    SlSkeleton,
-    SlSpinner,
-    SlSplitPanel,
-    SlSwitch,
-    SlTab,
-    SlTabGroup,
-    SlTabPanel,
-    SlTag,
-    SlTextarea,
-    SlTooltip,
-    SlTree,
-    SlTreeItem,
-    SlVisuallyHidden,
+    SlAlert as Alert,
```

Comment:
Q: why are we having to type all these using "As"? Aren't the types of all these components not automatically inferred?

Reply:
I used "as" in order to have all components with common names, without the "Sl" prefix.

I'm not sure if the final user will have the need to distinguish between pure Shoelace-based components and other ones.

If we use names without prefixes, we can replace any Shoelace component, like Button, with our own version if that's needed, or wrap them and add functionality, or add new components ...

Reply:
Fair point - it makes it easier for the end user at the expense of a little confusion for dev contributors - two competing concerns